<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Sudoku</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>depositphotos_108485338-stock-illustration-sudoku-flat-icon-with-long.jpg</normaloff>depositphotos_108485338-stock-illustration-sudoku-flat-icon-with-long.jpg</iconset>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;background-color: white;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="8">
     <spacer name="verticalSpacer_9">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4" rowspan="2" colspan="3">
     <layout class="QGridLayout" name="solutionLayout" columnstretch="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
     </layout>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="3" colspan="5">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">QLabel {


	text-align:center;
    font: 10pt &quot;Segoe Script&quot;;
    padding: 6px;

}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2" rowspan="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="8" rowspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="9">
     <spacer name="verticalSpacer_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="9">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3" rowspan="2">
     <widget class="QPushButton" name="prevDec">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton, QLineEdit {
    background-color: white;
    border-style: outset;
    border-width: 2px;

    border-color: rgb(0, 0, 0);
    font: 10pt &quot;Segoe Script&quot;;
    padding: 6px;

}

</string>
      </property>
      <property name="text">
       <string>Предыдущее
решение</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="7" rowspan="2">
     <widget class="QPushButton" name="nextDec">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton, QLineEdit {
    background-color: white;
    border-style: outset;
    border-width: 2px;

    border-color: rgb(0, 0, 0);
    font: 10pt &quot;Segoe Script&quot;;
    padding: 6px;
}

</string>
      </property>
      <property name="text">
       <string>Следующее
решение</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="9" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">QPushButton, QLineEdit {
    background-color: white;
    border-style: outset;
    border-width: 2px;

    border-color: rgb(0, 0, 0);
    font: 10pt &quot;Segoe Script&quot;;
  
    padding: 8px;
}</string>
        </property>
        <property name="text">
         <string>СГЕНЕРИРОВАТЬ
 ПОЛНОЕ СУДОКУ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="checkCells">
        <property name="styleSheet">
         <string notr="true">QPushButton, QLineEdit {
    background-color: white;
    border-style: outset;
    border-width: 2px;

    border-color: rgb(0, 0, 0);
    font: 10pt &quot;Segoe Script&quot;;
    min-width: 10em;
    padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>ПРОВЕРИТЬ ВСЕ ЯЧЕЙКИ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearField">
        <property name="styleSheet">
         <string notr="true">QPushButton, QLineEdit {
    background-color: white;
    border-style: outset;
    border-width: 2px;

    border-color: rgb(0, 0, 0);
    font: 10pt &quot;Segoe Script&quot;;
    min-width: 10em;
    padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>ОЧИСТИТЬ ПОЛЕ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decisions">
        <property name="styleSheet">
         <string notr="true">QPushButton, QLineEdit {
    background-color: white;
    border-style: outset;
    border-width: 2px;

    border-color: rgb(0, 0, 0);
    font: 10pt &quot;Segoe Script&quot;;
    min-width: 10em;
    padding: 6px;
}

</string>
        </property>
        <property name="text">
         <string>КОЛИЧЕСТВО РЕШЕНИЙ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generate">
        <property name="styleSheet">
         <string notr="true">QPushButton, QLineEdit {
    background-color: white;
    border-style: outset;
    border-width: 2px;

    border-color: rgb(0, 0, 0);
    font: 10pt &quot;Segoe Script&quot;;
    min-width: 10em;
    padding: 6px;
}
</string>
        </property>
        <property name="text">
         <string>СГЕНЕРИРОВАТЬ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="pbProgress">
        <property name="styleSheet">
         <string notr="true">QProgressBar {
    border: 2px solid black;
   font: 10pt &quot;Segoe Script&quot;;
	text-align: center;
 
}

QProgressBar::chunk {
  background-color: rgb(255, 163, 58)

}

</string>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="2">
     <layout class="QGridLayout" name="tabLayout" columnstretch="10"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
